src/bundle/Resources/views/themes/admin/account/settings/update.html.twig

```diff
@@ -1,41 +1,43 @@
-{% extends '@ibexadesign/ui/base.html.twig' %}
+{% set view_base_layout = '@ibexadesign/ui/layout.html.twig' %}
```

Comment:
Question: do we have to set it to override something? AFAICS `edit_base.html.twig` extend this by default. Could you explain what have I missed?

Reply:
Discussed in private: we are overriding the value provided to twig from kernel (core?), which does not have knowledge of `admin-ui`.